README.md

```diff
@@ -119,15 +119,27 @@ auth: OAuthTokenAuthStrategy = github.auth.as_web_user("<code>").exchange_token(
 access_token = auth.token
 refresh_token = auth.refresh_token
 # restore the user token from database
+
+# With  OAuthAPP
 user_github = github.with_auth(
     OAuthTokenAuthStrategy(
-        "<client_id>", "<client_secret>", refresh_token=refresh_token
+        "<client_id>", "<client_secret>", token=access_token 
+    )
+)
+# OR  with  GithubAPP
+user_github = github.with_auth(
+    OAuthTokenAuthStrategy(
+        "<client_id>", "<client_secret>", refresh_token=refresh_token 
     )
 )
 
 # now you can act as the user
 resp = user_github.rest.users.get_authenticated()
 user: PublicUser | PrivateUser = resp.parsed_data
+
+# you can get public email as
+email = user.email
+
```

Comment:
```suggestion

# you can get the user login id now
login_id = user.login
```

Reply:
Thank you for review.. I will make changes accordingly..

Reply:
> 

login_id = user.login  return login username , Is that we mean to say
login_id = user.id return Id , but for purpose this will be used ?

Reply:
user.login is the actual id for user. the user.id is the node id and has no meaning.

Reply:
@BhuwanPandey i have made a mistake on this. `user.id` is the actual id for user and the `user.node_id` is the node id actually.